File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.MoreEx_JS_Conditional_Stitiments-More Exercises/P07_FuelTank.js
function fuelTank(input) {
    let fuelType = input[0];
    let fuelQuantity = Number(input[1]);

    if (fuelType === "Diesel" || fuelType === "Gasoline" || fuelType === "Gas") {
        if (fuelQuantity < 25) {
            console.log(`Fill your tank with ${fuelType.toLowerCase()}!`);
        } else {
            console.log(`You have enough ${fuelType.toLowerCase()}.`);
        }
    } else {
        console.log("Invalid fuel!");
    }
}

fuelTank(["Diesel",
    10]
)

File: Java_Basic_Jan_2023_SolutionsOfProblems/P02AdditionalProblems_ConditionalStatements/P07_FuelTank.java
package P02AdditionalProblems_ConditionalStatements;

import java.util.Scanner;

public class P07_FuelTank {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String fuelType = scanner.nextLine();
        double fuelLtr = Double.parseDouble(scanner.nextLine());
        if ((fuelType.equals("Diesel")) || (fuelType.equals("Gasoline")) || (fuelType.equals("Gas"))) {
            if (fuelLtr >= 25)
                System.out.printf("You have enough %s.", fuelType.toLowerCase());
            else
                System.out.printf("Fill your tank with %s!", fuelType.toLowerCase());
        } else
            System.out.println("Invalid fuel!");


    }
}
